R/parameters.R

```diff
@@ -1,40 +1,55 @@
 #' @title Get model parameters
 #' @description
 #' get_parameters creates a named list of parameters for use in the model. These
-#' parameters are passed to process functions. These parameters are explained in
+#' parameters are passed to process functions. The default parameters, which
+#' are for Plasmodium falciparum, are explained in
 #' "The US President's Malaria Initiative, Plasmodium falciparum transmission
 #' and mortality: A modelling study."
+#' 
+#' Plasmodium vivax specific parameters are explained in
+#' "Mathematical modelling of the impact of expanding levels of malaria control
+#' interventions on Plasmodium vivax." by White, Michael T., et al.
 #'
 #' @param overrides a named list of parameter values to use instead of defaults
+#' @param parasite Plasmodium parasite species ("falciparum" or "vivax"); default = "falciparum"
+#' 
 #' The parameters are defined below.
 #'
 #' fixed state transitions:
 #'
 #' * dd - the delay for humans to move from state D to A; default = 5
```

Comment:
If expanding here to multiple parasites, do we want parasite-related defaults to be listed in this documentation of parameters, or should the csv be the reference to see defaults?

Reply:
I think we can change this later... if/when expanding can rethink as to how to document.